ui/src/ducks/app/monitoring.js

```diff
@@ -90,6 +93,9 @@ export function* handleClusterError(clusterHealth, result) {
   if (result.error.response) {
     yield put(setPrometheusApiAvailable(true));
     clusterHealth.error = `Prometheus - ${result.error.response.statusText}`;
+  } else if (clusterHealth.prometheusPodStatusMessage) {
+    yield put(setPrometheusApiAvailable(false));
+    clusterHealth.error = 'Unknown';
```

Comment:
No, actually if you want to have a different state in `clusterHealth` in case there's a missing Volume for Prometheus, then it's not "unknown", it's "missing a Volume". But I'd argue this can remain `prometheus_unavailable`, and the banner is enough to explain to the user **why** it is unavailable.